src/Business Foundation/App/NoSeriesCopilot/src/Copilot/NoSeriesCopilotImpl.Codeunit.al

```diff
@@ -61,7 +61,8 @@ codeunit 324 "No. Series Copilot Impl."
                 SaveGenerationHistory(NoSeriesGeneration, InputText);
                 CreateNoSeries(NoSeriesGeneration, GeneratedNoSeries, Completion);
             end else
-                ResponseText := Completion;
+                if Completion <> '' then
```

Comment:
What happens if `Completion` is empty? I'd expect this else check to just error irregardless of what Completion has.